### PR TITLE
NEW thirdparty in donation if option ON

### PR DESCRIPTION
**Pull request by ptibogxiv**

### CODE REVIEW

File: htdocs/don/card.php [hunk 1]
```diff
@@ -107,8 +107,8 @@
 		$object->amount      = price2num(GETPOST("amount",'alpha'));
 		$object->town        = GETPOST("town",'alpha');
 		$object->zip         = GETPOST("zipcode",'alpha');
-        $object->country_id  = GETPOST('country_id', 'int');
-        $object->email       = GETPOST("email",'alpha');
+		$object->country_id  = GETPOST('country_id', 'int');
```
Comment (eldy) [inline htdocs/don/card.php]: 'int' for en email ?

Comment (ptibogxiv) [reply htdocs/don/card.php]: bad fix when i submit :/ 👎 fix done 👍

File: htdocs/don/class/don.class.php [hunk 2]
```diff
@@ -1,10 +1,11 @@
-<?php
+<<?php
```
Comment (eldy) [inline htdocs/don/class/don.class.php]: This looks strange to me ...

Comment (ptibogxiv) [reply htdocs/don/class/don.class.php]: Bad fixe again 😣 but it's fixed now

**Pull request merged**